solr/solr-ref-guide/modules/configuration-guide/pages/requestdispatcher.adoc

```diff
@@ -107,45 +81,11 @@ This `HttpServletRequest` is not used by any Solr component, but may be useful w
 
 [source,xml]
 ----
-<requestParsers enableRemoteStreaming="false"
-                enableStreamBody="false"
-                multipartUploadLimitInKB="2048"
+<requestParsers multipartUploadLimitInKB="2048"
                 formdataUploadLimitInKB="2048"
                 addHttpRequestToContext="false" />
 ----
 
-The below command is an example of how to enable RemoteStreaming and BodyStreaming through the xref:config-api.adoc#commands-for-common-properties[Config API]:
```

Comment:
I prefer to not show any example on how to set any of these requestParsers details dynamically, thus I did not swap out the examples for the other still-valid configuration attributes.  IMO none of these should be in solrconfig.xml in the first place -- they are obscure things that I don't think someone would want to set on any one config (collection); instead I think all of them should be global.  Making stuff configurable has some maintenance cost and are arcane details to many people.  This is just my opinion.